.gitignore

```diff
@@ -11,3 +11,6 @@
 /.eunit
 .elixir.plt
 erl_crash.dump
+
+# Ingore IDE configs
```

Comment:
I think IDE-specific files should be handled by the global `.gitignore` file on the machine and not by the Elixir project.

https://help.github.com/articles/ignoring-files/

Reply:
Ok, i can remove revert it. I added it out of habit